### DISCUSSION TITLE
demo-restful-api version 0.3 in HTTP APIs not found in Docker Hub

### DISCUSSION THREAD

**Discussion by hugoscavino** [open-policy-agent/community#203] [OPA and Rego] [answered]
Hello. Where can I find the 0.3 version of the demo-restful-api image mentioned in the docker-compose.yml? When I traverse to Docker Hub there is only 0.2.
  
  https://hub.docker.com/r/openpolicyagent/demo-restful-api/tags
  
Thank you for the help

docker-compose.yml:
...
  api_server:
    image: openpolicyagent/demo-restful-api:0.3

**Comment by anderseknert** [comment, marked as answer]
Hiu @hugoscavino! Yeah, that's indeed rather outdated.. where is the docker compose file you're referring to? I guess we'll need to push an updated version.

This seems related: https://github.com/open-policy-agent/contrib/pull/176/files

CC @thecrazyrussian

**Reply by hugoscavino** [reply]
Thank you yes! That discussion is right on target.

https://github.com/open-policy-agent/contrib/pull/176

**Reply by anderseknert** [reply]
👍 https://github.com/open-policy-agent/contrib/pull/178
